controllers/IssueController.js

```diff
@@ -398,6 +442,7 @@ const validateData = (title, description, section, scope) => {
       .regex(/\w+/)
       .rule({ message: "Please provide a valid description" })
       .required(),
+    images: Joi.array().max(3).required(),
```

Comment:
Images arent `required` right?

Reply:
They are required. Even if there are jo images, an empty array must be passed to maintain consistency in database schema

Reply:
Okay